strings/trainer_sliding_strings.string

```diff
@@ -1,41 +1,202 @@
 #include "src/config.h"
 
-//Notice the extra whitespace character after the "\p"s. Without it, the message won't disappear from the screen when the player presses A.
+// Notice the extra whitespace character after the "\p"s. Without it, the message won't disappear from the screen when the player presses A.
+// Note the 34 character limit per line!
+
+// The following replacement text is used:
+//      [FD][06] - The pokemon's name (taken from original Unbound text)
 
 #org @gText_DefaultTrainerDynamaxMsg
 Time to Dynamax!\p 
 
 #org @gText_TestTrainerDynamaxMsg
 I now shall unleash the power\nof Dynamax!\p 
 
-//For Pokemon Unbound - Feel free to remove, and do not place strings below
-#ifdef UNBOUND
+// Rival 1
+#org @sText_Rival1_LastSwitchIn
+Okay, [FD][06]! Let's do this!\p 
+
+#org @sText_Rival1_LowHP
+No way! [FD][06] and I\naren't done yet!\p 
+
+// Rival n here!
+
+// Leader 1
```

Comment:
could make the comment like
// Gym Leader 1 (Terrence) (here as well as other places for the other leaders)